src/rust/iced-x86/src/instruction.rs

```diff
@@ -2346,11 +2345,11 @@ impl Instruction {
 	/// ```
 	#[cfg_attr(has_must_use, must_use)]
 	#[cfg_attr(feature = "cargo-clippy", allow(clippy::missing_inline_in_public_items))]
-	pub fn virtual_address<F>(&self, operand: u32, element_index: usize, get_register_value: F) -> u64
+	pub fn virtual_address<F>(&self, operand: u32, element_index: usize, mut get_register_value: F) -> Option<u64>
```

Comment:
This is a breaking change so I want to keep the original method name and just add a new one that returns `Option<u64>`. The original method can call that new method and `unwrap()` the result.

Reply:
Fair enough!